### ISSUE TITLE
Can you give more intuition on why id-mrf loss is used?

### ISSUE THREAD

**Issue by heyoon01** [YadiraF/DECA#33]
Hi! 
Thanks for sharing your wonderful work.
Hope to test out the training code someday.

I see in Figure 7 that the effect of L_mrf is quite crucial in obtaining mesh with details.
Can you share more intuition on why you chose to use id-mrf loss?

Id-mrf loss seems to be from an inpainting literature, 
and I don't get the need for finding the corresponding nearest neighbor feature patches when in the case of this paper,
the rendered and input images are supposed to be aligned in the image level.
Is this to handle some misalignments?
Was it better than using perceptual losses?
Thanks.

**Comment by TimoBolkart** [comment]
Hi, please see the [updated paper](https://files.is.tue.mpg.de/black/papers/SIGGRAPH21_DECA.pdf). We added more descriptions for each loss function. Does that answer your questions?

**Comment by heyoon01** [comment]
Thank you for the reply. Really appreciate it.
